torchvision/prototype/transforms/functional/_misc.py

```diff
@@ -39,4 +39,6 @@ def gaussian_blur_image_tensor(
 
 
 def gaussian_blur_image_pil(img: PIL.Image, kernel_size: List[int], sigma: Optional[List[float]] = None) -> PIL.Image:
-    return to_pil_image(gaussian_blur_image_tensor(to_tensor(img), kernel_size=kernel_size, sigma=sigma))
+    t_img = pil_to_tensor(img)
+    output = gaussian_blur_image_tensor(t_img, kernel_size=kernel_size, sigma=sigma)
+    return to_pil_image(output, mode=img.mode)
```

Comment:
This looks good but shouldn't we also do this on the functional on main area?
https://github.com/pytorch/vision/blob/095437aad6cfd25b01fa01961e5774904d738347/torchvision/transforms/functional.py#L1341-L1343